src/cmds/riichidb.ts

```diff
@@ -151,6 +151,53 @@ export class RiichiDbCommand implements CommandBuilder {
             const gamedata = new AttachmentBuilder(Buffer.from(parse(data[0]), 'utf-8')).setName("DataGame.csv");
             const playerdata = new AttachmentBuilder(Buffer.from(parse(data[1]), 'utf-8')).setName("DataPlayer.csv");
             event.reply({ files: [playerdata, gamedata] });
+        } else if (args[0] === 'verify') {
+            const entire_db = await RiichiDatabase.getEntireDB();
+            const gamedata_sqltbl = entire_db[0] as DataGameSQLEntry[];
+            const playerdata_sqltbl = entire_db[1] as DataPlayerSQLEntry[];
+
+            const adj_map: Map<string, number> = new Map();
+            const raw_map: Map<string, number> = new Map();
+            const games_played_map: Map<string, number> = new Map();
+            const rank_total_map: Map<string, number> = new Map();
+
+            gamedata_sqltbl.forEach((e) => {
+                adj_map.set(e.id_player_1, e.score_adj_1 + (adj_map.get(e.id_player_1) ?? 0));
```

Comment:
im not a fan of this implementation, its very verbose. at the very least, please document it better 👍